tests/RPCServer.test.ts

```diff
@@ -1000,6 +986,36 @@ describe(`${RPCServer.name}`, () => {
       // Shorter timeout is updated
       const ctxShort = await ctxShortProm.p;
       expect(ctxShort.timer.delay).toEqual(25);
+    }
+  });
+  test('handler overrides timeout - greater value', async () => {
+    {
+      const waitProm = promise();
+      const ctxLongProm = promise<ContextTimed>();
+      class TestMethodLongTimeout extends UnaryHandler {
+        // This will override the default timeout from 50 to 250, thereby increasing it.
+        handlerTimeout = 250;
```

Comment:
Ok so the handler will override the global server timeout. This is correct.

Reply:
But what about the `RPCClient`, shouldn't that also have a calling timeout override?

Reply:
It does, for overwriting streamKeepAliveTimeoutTime, but as we previously decided, client timeouts are only advisory, and play no real role

Reply:
Client timeouts are not advisory. Handler timeouts are. Client timeouts are strict.

Can you make sure these timeouts are named consistently?

Reply:
I need separate tests for these too. Check if there are existing tests, if not, add new tests, or replace old ones. Refactor the old code too. And read all pre-existing tests.

Reply:
Added jests